scripts/bountiful.ash

```diff
@@ -339,6 +339,121 @@ string addBountyToQueue(monster opp, boolean speculate) {
   return "";
 }
 
+/**
+* Determines if desired monster is in our received fax.
+* If it is not, send fax to clear inventory
+* @param {monster} enemy - desired fax target
+* @returns {boolean} - true if have photocopy of desired monster
+*/
+boolean checkFax(monster enemy)
```

Comment:
this is (generally) fine logic. i have one small quibble. what if a user is in a clan that actually is the clan that a monster is being saved in for the fax network? this isn't any real problem for any user in, say, BAFH (which i suspect is like 90-95% of users) but it may be worth noting in docs somewhere that you probably should be in BAFH or a similar clan for bountiful. folks should anyway, tbh!

Reply:
Updated wording